src/renderer/extensions/rule-editor/components/rule-list/whitelist-domain.ts

```diff
@@ -0,0 +1,16 @@
+export const WHITELIST_DOMAINS = [
+    'alilang-desktop-client.cn-hangzhou.log.aliyuncs.com',
+    's-api.alibaba-inc.com',
+    'alilang.alibaba-inc.com',
+    'auth-alilang.alibaba-inc.com',
+    'mdm-alilang.alibaba-inc.com',
+    '***.apple.com',
+    '*.mzstatic.com',
+    '*.cdn-apple.com', 
+    '***.apple-cloudkit.com',
+    '***.icloud.com',
+    '***.icloud-content.com',
+    '***.icloud.com.cn',
+    'txy.live-play.acgvideo.com',
+    'ocs-oneagent-server.alibaba.com',
+].join(' ');
```

Comment:
```suggestion
];
```

开一个 CONSTANTS 目录存放这类常量，我觉得这边存数组更好，在用的地方`join()`